wikimedia/wikipedia-ios
# Allow tapping diff header title to visit associated page

**Phabricator:** https://phabricator.wikimedia.org/T347915 (partially)

### Notes
Adds "hyperlink" to view associated article from the diff/revision view.

### Test Steps
1. Open the diff/revision view (from a Watchlist revision or article history)
2. Confirm title appears in theme's `link` color
3. Confirm tapping the title label navigates user to associated article
